src/dymaptic.GeoBlazor.Core/Scripts/jsBuilder.ts

```diff
@@ -585,7 +622,7 @@ export async function buildJsPopup(dotNetPopup: any, viewId: string): Promise<Po
         autoCloseEnabled: dotNetPopup.autoCloseEnabled ?? false,
         autoOpenEnabled: dotNetPopup.autoOpenEnabled ?? true,
         collapseEnabled: dotNetPopup.collapseEnabled ?? true,
-        collapsed: dotNetPopup.collapsed ?? false,
+        //collapsed: dotNetPopup.collapsed ?? false,
```

Comment:
hmm, this should probably be reverted? If it's causing problems, we can try `dotNetPopup.collapsed ?? undefined`.

Reply:
this looks like an artifact from a bad merge or something
<img width="815" alt="image" src="https://github.com/dymaptic/GeoBlazor/assets/126698247/2c52a1da-f89a-4ad8-a6ab-e317daf2ab1e">